### ISSUE TITLE
Emails from Microsoft Outlook ios app get parsed wrong

### ISSUE THREAD

**Issue by jfrazelle** [github/email_reply_parser#45]
So I may be making an incorrect assumption you all use this repo for also parsing the replies to issues and pull requests but I found this bug as visible here: https://github.com/docker/docker/issues/16847#issuecomment-146660607

Thanks!

**Comment by deanperry** [comment]
Yep, this has been happening for a while now :(  Hope it's sorted soon!

**Comment by CestLou** [comment]
+1 for seeing this issue solved.

**Comment by jsatk** [comment]
Any progress on this issue? Would love to be able to reply to PRs from Outlook for iOS without it posting malformed replies.

**Comment by aoozdemir** [comment]
This is also happening on Spark and Thunderbird (MacOS).
